packages/case-study-slider/index.js

```diff
@@ -1,229 +1,38 @@
-import React, { Component } from 'react'
-import Logo from './Logo'
-import StatusBar from './StatusBar'
-import Button from '@hashicorp/react-button'
+import React from 'react'
 import Image from '@hashicorp/react-image'
 import fragment from './fragment.graphql'
-
-class CaseStudySlider extends Component {
-  constructor(props) {
-    super(props)
-    this.data = this.props.data
-    const timing = this.props.timing ? parseInt(this.props.timing) : 10
-    this.state = {
-      active: 0,
-      timing: timing,
-      numFrames: this.data.caseStudies.length,
-      measure: true,
-      containerWidth: 0,
+import { FeaturedSliderInner } from '@hashicorp/react-featured-slider'
```

Comment:
`CaseStudySlider` and `FeaturedSlider` were nearly identical. Rather than rewrite both in CSS modules, I've rewritten `FeaturedSlider`, added a named export `FeaturedSliderInner`, and have refactored `CaseStudySlider` to massage data and use `FeaturedSliderInner`.